main.js

```diff
@@ -0,0 +1,34 @@
+const csv = require('csv-parser');
+const fs = require('fs');
+const results = []; //to store the result of the csv file here!
+
+function readPlain () {
+	fs.createReadStream('input.csv')
+	.pipe(csv({}))
+	.on('data', (data) => results.push(data))
+	.on('end', () => {
+		console.log(results);
+	}
+	);
+}
+
+function saveToFile (usersArr) {

```

Comment:
This supposed to be a reusable piece of code, means, we shall not hard code any values. 
`function saveToFile(outputFileName="output.json", encoding="utf-8", data)
     if (!data) return;
     ...
`